index.php

```diff
@@ -295,7 +295,8 @@
 }
 
 define('VIEWPATH', $view_folder . DIRECTORY_SEPARATOR);
-
+// Define se vai ser produção ou ambiente de testes
+define('ENABLE_SENDBOX', true);
```

Comment:
Sugestão: Eu acho que o nome da variável poderia ser: 

```suggestion
define('PAGAMENTOS_MODO_SANDBOX', true);
```